plugins/nanospell/plugin.js

```diff
@@ -61,13 +62,17 @@
 			// non-root block nodes must also be excluded.
 			// the text content of ckeditor bookmarks must also be excluded
 			// or &nbsp; will be added throughout.
+
 			var path = new CKEDITOR.dom.elementPath(node, startNode);
 
-			return node.type == CKEDITOR.NODE_TEXT && // it is a text node
+			var condition = node.type == CKEDITOR.NODE_TEXT && // it is a text node
 				node.getLength() > 0 &&  // and it's not empty
 				( !node.isReadOnly() ) &&   // or read only
-				isNotBookmark(node) &&  // and isn't a fake bookmarking node
-				(path.block && path.block.equals(startNode)); // it's not nested in a deeper block from our root.
+				isNotBookmark(node) && // and isn't a fake bookmarking node
+				(path.blockLimit ? path.blockLimit.equals(startNode): true) &&
+				(path.block ? path.block.equals(startNode): true); // check we don't enter nested blocks (special list case)
```

Comment:
ternary is required because it's just left null if you're not actually in a blockLimit or block.  not sure of what the cases are but whatever.  

the addition of path.blockLimit fixes tables.